package.json

```diff
@@ -23,6 +23,7 @@
     "apollo-link-context": "1.0.18",
     "apollo-link-error": "1.1.11",
     "apollo-link-http": "1.5.15",
+    "blockstack": "^19.2.1",
```

Comment:
We use fixed versions for main dependencies.

Reply:
updated!